device/pcie/pci_device.hpp

```diff
@@ -34,84 +34,144 @@ struct PciDeviceInfo
 };
 
 class PCIDevice {
+    const std::string device_path;  // Path to character device: /dev/tenstorrent/N
+    const int device_num;           // N in /dev/tenstorrent/N
+    const int logical_id;           // Unique identifier for each device in entire network topology
+    const int fd;                   // Character device file descriptor
```

Comment:
I really don't have a problem with longer and more descriptive variable names, now that we're out of the dark age and don't have such limits on variable names and code sizes, etc.
Can we name this pci_device_file_desc?

Reply:
Sure, I renamed it.